Toolchain/BuildIt.sh

```diff
@@ -230,12 +238,20 @@ pushd "$DIR/Build/"
         pushd "$BUILD"
             CXXFLAGS="-DBUILDING_SERENITY_TOOLCHAIN" cmake ..
             cmake --build . --target LibC
-            install -D Libraries/LibC/libc.a Libraries/LibM/libm.a Root/usr/lib/
+            "$INSTALL" -D Libraries/LibC/libc.a Libraries/LibM/libm.a Root/usr/lib/
             SRC_ROOT=$(realpath "$DIR"/..)
-            for header in "$SRC_ROOT"/Libraries/Lib{C,M}/**/*.h; do
-                target=$(echo "$header" | sed -e "s@$SRC_ROOT/Libraries/LibC@@" -e "s@$SRC_ROOT/Libraries/LibM@@")
-                install -D "$header" "Root/usr/include/$target"
-            done
+            if [ "$(uname)" = "Darwin" ]; then
+                FILES=`find "$SRC_ROOT"/Libraries/LibC "$SRC_ROOT"/Libraries/LibM -name '*.h' -print`
```

Comment:
Uhh, I've been trying to avoid conditioning anything on Apple or Darwin (and telling you about it the whole time). Let's support older Bash versions unconditionally, so that we have less code paths to maintain, and so that other people/systems that use older Bash can benefit from it.

Using `find` for this is a good idea; but please use `$()` instead of backticks, and name the variable something more descriptive (`HEADERS`, perhaps). Also I'm concerned about file names that contain spaces and other fun bytes, but guess that was already broken here.

Reply:
Would it be better to tell people on MacOS to install a more recent version of bash than 3.x via homebrew?

Reply:
We are already telling them to install Bash via homebrew, by the way, but we don't mention a version explicitly.

Reply:
I've been removing the "Darwin" branching. 
My understanding was that the conditioning on Apple or Darwin was to be avoided mainly on the build files and not on shell, also because there are already Apple/Darwin conditioning in files.
I agree anyway that it's much better to reduce them when technically possible like in this case!

Regarding @ADKaster consideration, I think it's easier to reduce the number of things that people need to install, in particular if it requires a few line changes. This script works also on zsh, that is default macOS shell from Catalina going forward.